DXMainClient/DXGUI/Multiplayer/LANLobby.cs

```diff
@@ -276,21 +283,26 @@ private void LanGameLoadingLobby_GameLeft(object sender, EventArgs e)
 
         private void WindowManager_GameClosing(object sender, EventArgs e)
         {
-            if (socket == null)
-                return;
-
-            if (socket.IsBound)
+            if (socket != null && socket.IsBound)
             {
                 try
                 {
-                    SendMessage("QUIT");
+                    SendMessage("QUIT ");
```

Comment:
Is this modification intended? From `"QUIT"` to `"QUIT "`

Reply:
THank you!